### PR TITLE
chore: migrate away from blue-build

### PR DESCRIPTION
**Pull request by HerrNaN**
This PR removes the dependency on `blue-build/github-action` and takes ownership of the build process. The changes include:
- Migrate the contents of `recipe.yaml` to a `Containerfile`
- Replace `blue-build/github-action` with base steps
- Cleanup unused remnants of blue-build

### CODE REVIEW

**Comment by HerrNaN** [comment]
image

**Comment by HerrNaN** [comment]
image
IT WORKS!!! :tada:

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -4,31 +4,76 @@ on:
     - cron: "00 17 * * *" # build at 17:00 UTC every day 
                           # (20 minutes after last ublue images start building)
   push:
+    branches:
+      - main
```
Comment (williamleven) [inline .github/workflows/build.yml:8]: Is this intentional?

**Review by williamleven — APPROVED** [review]
So freaking neat!

File: .github/workflows/build.yml (hunk 1 above)
Comment (HerrNaN) [reply .github/workflows/build.yml:8]: yes, branch builds will only be triggered on PRs, leaving push on for all branches causes double workflow triggers (pull_request and push) for pullrequests which is annoying

**Pull request merged**